client/src/pages/feedback.tsx

```diff
@@ -202,6 +302,30 @@ function FeedbackItem(props: {
             <IconButton onClick={() => onUpdateAnswer(undefined)}>
               <CloseIcon />
             </IconButton>
+
+            {accessToken ? (
```

Comment:
why not just also check for selectedAnswerID here aswell isntead of doing `|| ""` below